cclib/io/filewriter.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, ccdata, jobfilename=None, indices=None, terse=False,
         self.jobfilename = jobfilename
         self.indices = indices
         self.terse = terse
+        if "ghost" in kwargs:
+            self.ghost = kwargs["ghost"]
```

Comment:
A one-liner:
```python
self.ghost = kwargs.get("ghost")
```
which has the advantage of always making sure `hasattr(my_writer, "ghost")` is true, and only `None` needs to be checked for.